hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulator.java

```diff
@@ -6,8 +6,9 @@
 import com.hazelcast.spi.Operation;
 
 import java.util.Random;
-import java.util.concurrent.atomic.AtomicInteger;
 
+import static com.hazelcast.client.impl.protocol.util.BitUtil.CACHE_LINE_LENGTH;
```

Comment:
I think you can put `CACHE_LINE_LENGTH` into `com.hazelcast.nio.Bits` (leaving client one as is). It has already a field for integer size, `INT_SIZE_IN_BYTES`. I'll request client team to do additional changes.